### PR TITLE
Upgrade redux-observable typings to 0.17

### PR DESCRIPTION
**Pull request by lgeiger**
`redux-observable` exposes the `ofType` lettable operator since 0.17 https://github.com/redux-observable/redux-observable/pull/343

### CODE REVIEW

**Comment by GAntoine** [comment]
@lgeiger Instead of overwriting the definitions for 0.14, could you copy then to a 0.17 folder and made the changes there?

**Comment by lgeiger** [comment]
I addressed your comments. Sorry for the long wait, I forgot to push 😉

**Comment by rgbkrk** [comment]
What should we do for the tests / Travis here @GAntoine?

File: definitions/npm/redux-observable_v0.17.x/flow_v0.47.x-/redux-observable_v0.17.x.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import { Observable } from 'rxjs';
```
Comment (GAntoine) [inline definitions/npm/redux-observable_v0.17.x/flow_v0.47.x-/redux-observable_v0.17.x.js]: Move this import inside `declare module 'redux-observable'`

File: definitions/npm/redux-observable_v0.17.x/test_redux-observable_v0.17.x.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+type Action = { type: 'A' | 'B' };
+
+/*
+ * Epic type
+ */
+const epic1: Epic<State, Action, void> =
+  (action$) =>
+    action$.filter(action => action.type === 'A')
+      .mapTo({ type: 'B' });
+const epic2: Epic<State, Action, void | number> =
```
Comment (GAntoine) [inline definitions/npm/redux-observable_v0.17.x/test_redux-observable_v0.17.x.js]: I don't think this is valid... You might need to pick between `void` and `number`.

**Comment by loyd** [comment]
Any progress??

**Comment by villesau** [comment]
Looks like redux-observable is already in 1.0 version. I'm going to close this. Feel free to open a new PR @lgeiger if needed.

**Pull request closed without merging**